ludwig/schema/combiners/common_transformer_options.py

```diff
@@ -6,7 +6,7 @@
 from ludwig.schema.metadata.combiner_metadata import COMBINER_METADATA
 
 
-@dataclass
+@dataclass(repr=False)
```

Comment:
Since this is everywhere it might be worth defining a lambda in some utils file.